packages/main/src/StepInput.ts

```diff
@@ -133,11 +133,11 @@ type StepInputValueStateChangeEventDetail = {
 class StepInput extends UI5Element implements IFormInputElement {
 	/**
 	 * Defines a value of the component.
-	 * @default 0
+	 * @default ""
 	 * @public
 	 */
-	@property({ type: Number })
-	value = 0;
+	@property()
+	value = "";
```

Comment:
I would return this to `number = 0` since the value of this component is meant to be used as number

Reply:
The reason why I made it string, is because the native input with type="number" returns string as value.
<input type="number"  value="0"/> => $0.value = "0"

It's up for discussion, I don't mind it being either way, but if we want to be as close as we can to the native I would suggest to make it string.

Reply:
We've had a discussion with the team, we will keep it as value type number. It feels more natural a numberic input to have the value type number, and when you work with the value, to escape the casting before doing any calculations with the value.